projects/kit/components/data-list-wrapper/data-list-wrapper.component.ts

```diff
@@ -56,12 +59,17 @@ export class TuiDataListWrapperComponent<T> implements TuiDataListAccessor<T> {
 
     @Input()
     @tuiDefaultProp()
-    size: TuiSizeXS | TuiSizeL = 'm';
+    size: DataListWrapperOptions<T>['size'] = this.componentOptions.size;
+
+    constructor(
+        @Inject(TUI_DATA_LIST_WRAPPER_OPTIONS)
+        private readonly componentOptions: DataListWrapperOptions<T>,
```

Comment:
`options` instead of `componentOptions`

Reply:
Sure, but what about `private readonly options: QueryList<TuiOptionComponent<T>> = EMPTY_QUERY;` on line 46?

Reply:
oh, sorry)

Reply:
but you can rename options to `queryOptions`

Reply:
Seems good